rogue-thi-app/pages/rooms/suggestions.jsx

```diff
@@ -309,7 +309,11 @@ export default function RoomSearch () {
                       </Link>
                       {TUX_ROOMS.includes(roomResult.room) && <> <FontAwesomeIcon title="Linux" icon={faLinux} /></>}
                       <div className={styles.details}>
-                        {getTranslatedRoomFunction(roomResult.type)}
+                        {
+                          roomResult.capacity
+                            ? `${getTranslatedRoomFunction(roomResult.type).split('(')[0]} (${roomResult.capacity} ${t('rooms.suggestions.seats')})`
```

Comment:
Since this line occurs three times across all room pages, it could be merged into a helper function for the code style and quality